src/main/java/emissary/util/WindowedSeekableByteChannel.java

```diff
@@ -91,7 +91,7 @@ private void realignBuffers() throws IOException {
             return;
         }
         // keep track of our position
-        final int offset = this.buff1.position() + this.buff2.position();
```

Comment:
This is not the original line that was impacted, but converting an addition variable to a long made more sense to me than casting in-line during a subtraction operation.